### PR TITLE
Clean up manifests

### PR DESCRIPTION
**Pull request by hugohills-regnosys**

### CODE REVIEW

File: com.regnosys.rosetta.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -5,8 +5,8 @@ Bundle-Name: com.regnosys.rosetta.tests
 Bundle-SymbolicName: com.regnosys.rosetta.tests;singleton:=true
 Bundle-Version: 0.0.0.master
 Bundle-Vendor: REGnosys
-Require-Bundle: org.junit.jupiter.api;bundle-version="[5.1.0,6.0.0)",
- org.eclipse.xtext.xbase.lib;bundle-version="2.25.0",
+Require-Bundle: org.junit.jupiter.api,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
```
Comment (SimonCockx) [inline com.regnosys.rosetta.tests/META-INF/MANIFEST.MF:9]: Shouldn't this one be on 2.27.0? (same as Xtext)

File: com.regnosys.rosetta.ui/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -11,13 +11,13 @@ Require-Bundle: com.regnosys.rosetta,
...
- org.eclipse.ui.ide;bundle-version="3.18.0",
+ org.eclipse.ui.editors,
+ org.eclipse.ui.ide,
  org.eclipse.ui,
  org.eclipse.compare,
  org.eclipse.xtext.builder,
- org.eclipse.xtext.xbase.lib;bundle-version="2.25.0",
- org.eclipse.xtend.lib;bundle-version="2.25.0";resolution:=optional,
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
+ org.eclipse.xtend.lib;bundle-version="2.14.0",
```
Comment (SimonCockx) [inline com.regnosys.rosetta.ui/META-INF/MANIFEST.MF:20]: Same with these two

File: com.regnosys.rosetta/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -15,14 +15,14 @@ Require-Bundle: com.google.guava,
...
  org.eclipse.emf.ecore;visibility:=reexport,
- org.eclipse.equinox.common;bundle-version="3.15.100",
+ org.eclipse.equinox.common,
  org.eclipse.xtend.lib.macro,
- org.eclipse.xtend.lib;bundle-version="2.25.0",
+ org.eclipse.xtend.lib;bundle-version="2.14.0",
  org.eclipse.xtext.util,
  org.eclipse.xtext.xbase,
- org.eclipse.xtext.xbase.lib;bundle-version="2.25.0",
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
```
Comment (SimonCockx) [inline com.regnosys.rosetta/META-INF/MANIFEST.MF:23]: ... and these two...

File: com.regnosys.rosetta.tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (hugohills-regnosys) [reply com.regnosys.rosetta.tests/META-INF/MANIFEST.MF]: Updated

File: com.regnosys.rosetta.ui/META-INF/MANIFEST.MF (hunk 2 above)
Comment (hugohills-regnosys) [reply com.regnosys.rosetta.ui/META-INF/MANIFEST.MF]: Updated

File: com.regnosys.rosetta/META-INF/MANIFEST.MF (hunk 3 above)
Comment (hugohills-regnosys) [reply com.regnosys.rosetta/META-INF/MANIFEST.MF]: Updated

**Pull request merged**